transaction-status/src/parse_associated_token.rs

```diff
@@ -11,8 +11,12 @@ use {
 
 // A helper function to convert spl_associated_token_account::id() as spl_sdk::pubkey::Pubkey
 // to solana_pubkey::Pubkey
+#[deprecated(
+    since = "2.3.0",
+    note = "Use `spl_associated_token_account::id()` instead"
```

Comment:
It works either way, but the smaller crate with this ID is `spl_associated_token_account_client::program::id()`, so I'd prefer recommending that instead

Reply:
I see this one is a re-export so works for me:
https://github.com/solana-program/associated-token-account/blob/109de0bf04dc033873941c6befed2a7ab07a93d9/program/src/lib.rs#L26-L28